main/forms.py

```diff
@@ -22,3 +28,18 @@ def save(self, commit: bool = True) -> User:
         if commit:
             user.save()
         return user
+
+
+class CustomUpdateUserForm(ModelForm):  # type: ignore
```

Comment:
This mypy error can be fixed with
```suggestion
class CustomUpdateUserForm(ModelForm[User]):
```